.github/workflows/test.yml

```diff
@@ -27,5 +27,5 @@ jobs:
         run: flake8 app/
       - name: Run tests
         timeout-minutes: 5
-        run: pytest tests/
+        run: pytest
```

Comment:
Why need to change the test run directory?

Reply:
It was a temporary solution that I forgot to remove